src/subframe_decoding.jl

```diff
@@ -381,3 +388,195 @@ function create_data(
 end
 
 
+function create_data_sub1(
+    TLM_HOW_Data::TLM_HOW_Data_Struct,
+    subfr_1_data::Subframe_1_Data)
+    
+
+    data = GPSData(
+        TLM_HOW_Data.integrity_status_flag,
+        TLM_HOW_Data.TOW,
+        TLM_HOW_Data.alert_flag,
+        TLM_HOW_Data.anti_spoof_flag,
+
+        subfr_1_data.trans_week,
+        subfr_1_data.codeonl2,
+        subfr_1_data.ura,
+        subfr_1_data.svhealth,
+        subfr_1_data.IODC,
+        subfr_1_data.l2pcode,
+        subfr_1_data.T_GD,
+        subfr_1_data.t_oc,
+        subfr_1_data.a_f2,
+        subfr_1_data.a_f1,
+        subfr_1_data.a_f0,
+
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing,
+        nothing
+    )
```

Comment:
```suggestion
data = GPSData(
      prev_data,
      integrity_status_flag = TLM_HOW_Data.integrity_status_flag,
      TOW = TLM_HOW_Data.TOW,
      alert_flag = TLM_HOW_Data.alert_flag,
      anti_spoof_flag = TLM_HOW_Data.anti_spoof_flag,
      IODE_Sub_2 = subfr_2_data.IODE,
      C_rs = subfr_2_data.C_rs,
      Δn = subfr_2_data.Δn,
      M_0 = subfr_2_data.M_0,
      C_uc = subfr_2_data.C_uc,
      e = subfr_2_data.e,
      C_us = subfr_2_data.C_us,
      sqrt_A = subfr_2_data.sqrt_A,
      t_oe = subfr_2_data.t_oe,
      fit_interval = subfr_2_data.fit_interval,
      AODO = subfr_2_data.AODO,
)
```
Uh I didn't know, that I can do this on Github ;)